### PR TITLE
Added visualization of the NURBS

### PR DESCRIPTION
**Pull request by maik-nack**
### Changes
- Wrote code for recursively plotting of the NURBS curve
- Added visualization's output in *.mp4* and *.gif* formats

### CODE REVIEW

File: nurbs.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        plt.close(fig)
+    progress(total, total)
+
+    out_mp4.close()
+    out_gif.close()
+
+
+if __name__ == '__main__':
+    # x, y, weight
+    points = np.array([
```
Comment (daniilsavchuk) [inline nurbs.py:101]: How do you build the knots?

**Review by daniilsavchuk — APPROVED** [review]
Please provide exclamation and go ahead with merging

File: nurbs.py (hunk 1 above)
Comment (maik-nack) [reply nurbs.py:101]: I use all control points for plotting a curve.  
So, the knot vector is `(0,0,...,0,1,1,...1)` where `#0 = #1 = order`.

**Comment by daniilsavchuk** [comment]
Approved, great job!

**Pull request merged**